src/Common/nunit/ConsoleOptions.cs

```diff
@@ -103,14 +103,14 @@ public ConsoleOptions(params string[] args)
                 v => ActiveConfig = RequiredValue(v, "--config"));
 
             // Where to Run Tests
-            this.Add("process=", "{PROCESS} isolation for test assemblies.\nValues: Single, Separate, Multiple",
+            this.Add("process=", "{PROCESS} isolation for test assemblies.\nValues: Single, Separate, Multiple. If not specified, defaults to Separate for a single assembly or Multiple for more than one.",
                 v => ProcessModel = RequiredValue(v, "--process", "Single", "Separate", "Multiple"));
 
-            this.Add("domain=", "{DOMAIN} isolation for test assemblies.\nValues: None, Single, Multiple",
+            this.Add("domain=", "{DOMAIN} isolation for test assemblies.\nValues: None, Single, Multiple. If not specified, defaults to Separate for a single assembly or Multiple for more than one.",
                 v => DomainUsage = RequiredValue(v, "--domain", "None", "Single", "Multiple"));
 
             // How to Run Tests
-            this.Add("framework=", "{FRAMEWORK} type/version to use for tests.\nExamples: mono, net-3.5, v4.0, 2.0, mono-4.0",
+            this.Add("framework=", "{FRAMEWORK} type/version to use for tests.\nExamples: mono, net-3.5, v4.0, 2.0, mono-4.0. If not specified, tests will be run under the current framework.",
```

Comment:
Actually, defaults to running under the framework version for which the assembly was built. If memory serves, it now only uses the current framework if the specified or default framework is not installed.